ai-catalog/samples/genai-summarization/src/main/java/com/android/ai/samples/genai_summarization/GenAISummarizationScreen.kt

```diff
@@ -51,14 +50,13 @@ fun GenAISummarizationScreen(viewModel: GenAISummarizationViewModel = hiltViewMo
     val sampleTextOptions = stringArrayResource(R.array.summarization_sample_text)
 
     val sheetState = rememberModalBottomSheetState()
-    var showBottomSheet by remember { mutableStateOf(false) }
-    val context = LocalContext.current
 
-    val summarizationResult = viewModel.summarizationGenerated.collectAsState()
+    val uiState by viewModel.uiState.collectAsStateWithLifecycle()
     var textInput by remember { mutableStateOf("") }
 
     Scaffold(
-        modifier = Modifier.fillMaxSize(), topBar = {
+        modifier = Modifier.fillMaxSize(),
+        topBar = {
             TopAppBar(
```

Comment:
While you are in this file, can you please swap the titleContentColor to onPrimaryContainer? primary is meant as a background color, that's why it is blue on blue